src/pages/gd/Swap/SwapCore/index.tsx

```diff
@@ -50,6 +52,35 @@ const SwapCore = memo(() => {
         value: '',
     })
 
+    const [pairData, setPairData] = useState()
+
+    useEffect(() => {
+        const getPairData = async () => {
+            if (Number(chainId) === SupportedChainId.FUSE) {
+                let pairData: any
+                const latestPairData = await AsyncStorage.getItem('voltage_latest_pair_data')
+                const recentPairData = latestPairData ? JSON.parse(latestPairData) : null
+
```

Comment:
it's possible to avoid let here:

```typescript
const pairData = recentPairData ? undefined : await voltagePairData()
```

Reply:
i dont understand why caching is done here and not in the sdk itself
in the sdk itself you count on the code here for cache.
in anycase this is infinite cache, we dont need it. see my comment in the other PR.
this PR can probably be closed